lib/identity/client.js

```diff
@@ -766,6 +766,51 @@ class Client extends PartialClient {
     )
     return await validateResponseAsJSON(response)
   }
+  /**
+   * removeSingleSecret
+   * @param {String} secretID
+   * @param {String} version
+   * @param {String} secretName
+   * @param {String} secretType
+   * @return {Boolean} returns true if the secret was successfully removed
+   */
+  async removeSingleSecret(secretID, version, secretName, secretType) {
```

Comment:
keep the naming consistent, its `createSecret` so I expect `deleteSecret`, remove would be congruent if there was an add method

Reply:
and for single, maybe `deleteSecretVersion`

Reply:
only pushback here is at some point we do need to differentiate between a secret version, and an entire secret (all versions). Which get removeSecret and which has a different name?

Reply:
clearly didn't refresh my page and see you had already said what I repeated